frontend/src/components/Roadmap/PhaseCard.jsx

```diff
@@ -1,12 +1,12 @@
 import React, { useState } from 'react';
 import MilestoneCard from './MilestoneCard';
 import { ChevronDown, ChevronRight, Calendar, Clock } from 'lucide-react';
-import { getPhaseColor } from '@/utils/roadmapUtils';
+import { getPhaseColor, calculatePhaseProgress } from '@/utils/roadmapUtils';
 
 /**
  * PhaseCard Component
  * 
- * Displays a project phase with milestone expansion functionality.
+ * Displays a project phase with milestone expansion functionality and real progress tracking.
```

Comment:
same here. "Displays a project phase with milestone expansion functionality and progress tracking" would be good